components/NewEvent.tsx

```diff
@@ -63,40 +59,14 @@ class NewEvent extends React.Component<Props, State> {
       creator: this.props.currentUserLimited
     }
   }
-  
 
-  getPermission = async () => {
```

Comment:
So we don't need any permission? Or this was moved elsewhere to be called from all upload calls?

Reply:
It was moved to be called from all upload calls :)